src/txmempool.cpp

```diff
@@ -798,6 +798,16 @@ bool CTxMemPool::lookup(uint256 hash, CTransaction& result) const
     return true;
 }
 
+bool CTxMemPool::lookup(uint256 hash, CTransaction& result, int64_t& time) const
```

Comment:
Have you considered adding `int64_t& time` to the previous `lookup`? Both functions are almost the same...